.circleci/config.yml

```diff
@@ -207,11 +211,14 @@ jobs:
           name: Install build dependencies
           command: |
             apt-get -qq update
-            apt-get -qy install python-sphinx python-pip
+            apt-get -qy install python-sphinx python-pip codespell
       - run: *setup_prerelease_commit_hash
       - run:
           name: Build documentation
           command: ./scripts/docs.sh
+      - run:
+          name: Check spelling
```

Comment:
I don't think we should run this twice (on linux and mac). Actually it would be great if it could be moved to its own job (like the `docs` job) and as such would show up separately on github.

Reply:
All right, will be moved to a separate job.

Reply:
@axic done. I moved it to a separate job. Currently it is failing and it is supposed to fail, please see https://circleci.com/gh/ethereum/solidity/22005?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link as I pulled out the two changes into https://github.com/ethereum/solidity/pull/4464 , which should be merged first.

Reply:
Did you mean to remove this? Does it something else than the job above?

Reply:
@chriseth Removed as well. Only a separate job remains now.